samples/Exporters/Console/TestConsoleActivity.cs

```diff
@@ -27,7 +27,7 @@ internal static object Run(ConsoleActivityOptions options)
         {
             // Enable OpenTelemetry for the source "MyCompany.MyProduct.MyWebServer"
             // and use Console exporter
-            OpenTelemetrySdk.Default.EnableOpenTelemetry(
+            OpenTelemetrySdk.EnableOpenTelemetry(
```

Comment:
This one won't be disposed. Should be (something like) `using var openTelemetry = OpenTelemetrySdk.EnableOpenTelemetry(` no?